Wireframe/css/style.css

```diff
@@ -0,0 +1,164 @@
+:root {
+  --font-size: sans-serif;
+  --dark-orange: #c05326;
+  --light-gray: #dad7cd;
+  --slate-dark: #4c5760;
+  --light-blue: #bcb8b1;
+}
+
+body {
+  color: var(--slate-dark);
+  margin: 5px;
+  font-size: 18px;
+  background-color: rgb(244, 244, 240);
+  font-family: sans-serif;
+}
+
+.gitImage {
+  display: grid;
+  height: 150px;
+  width: 150px;
+  justify-content: center;
+  align-content: center;
+  border-radius: 75px;
+  opacity: 0.7;
+}
+
+.gridContainer {
+  height: 100%;
+  display: grid;
+  grid-gap: 20px;
+  padding: 20px;
+  grid-template-columns: repeat(2, 1fr);
+  grid-template-rows: 200px 400px 300px 80px;
+  grid-template-areas:
```

Comment:
Everything you write as code should make things clearer, not harder to understand. Good code is clear and simple. 

Don't use letters, use the actual words. There's no benefit to using a letter - it obscures your meaning for other developers, and in production all your code is automatically compressed and minified, so you're not saving any bytes this way.